go/badges/badgestate.go

```diff
@@ -197,7 +198,7 @@ func (b *BadgeState) UpdateWithGregor(ctx context.Context, gstate gregor.State)
 	b.state.RevokedDevices = []keybase1.DeviceID{}
 	b.state.NewTeams = nil
 	b.state.DeletedTeams = nil
-	b.state.NewTeamAccessRequests = nil
+	b.state.NewTeamAccessRequests = 0
```

Comment:
how would you feel about renaming this to something like NewTeamAccessRequestsCount since almost all of the other things are actually what they hold

Reply:
Will do